remotespark/utils/constants.py

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the Modified BSD License.
 
 
-class Constants:
+class Constants(object):
     config_json = "config.json"
```

Comment:
This should have been corrected a long time ago, but this shouldn't be a class either.  The class definition should be factored out.  We can take care of that in a later CR.